B827EBFFFE2F09A5.json

```diff
@@ -0,0 +1,18 @@
+{ 
+	"gateway_conf": { 
+		"gateway_ID": "B827EBFFFE2F09A5", 
+		"servers": [ 
+			{ 
+				"server_address": "router.sl2.thethings.network", 
```

Comment:
I think you have a typo in the url, `router.sl2.thethings.network` doesn't resolve to anything.